webapp/client/cypress/integration/logout.spec.js

```diff
@@ -0,0 +1,20 @@
+describe("Logout", () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit("logout");
+  });
+
+  it("has no back link", () => {
+    cy.get("a").contains("Zurück").should("not.exist");
+  });
+
+  it("logout to login page", () => {
+    cy.get("button[type=submit]").contains("Abmelden").click();
+
+    cy.url().should("include", "unlock_code_activation/step/data_input");
+
+    cy.get("div[class*=alert-success]").contains(
+      "Sie haben sich erfolgreich abgemeldet."
```

Comment:
I am wondering if there would be a way to ensure that a completed tax return user is directly sent to the success page. If any one of you can think of an easy way to do this, nice. Otherwise, it is fine like that.

Reply:
We could just follow the ui steps after login and send the tax declaration so that the user.pdf is set?

Reply:
We talked about it and decided to have a test that sets the data via the testing route and then just sends the tax declaration.